src/form/form-numeric-input/form-numeric-input.styles.ts

```diff
@@ -0,0 +1,13 @@
+import { createUseStyles } from '../../styles/create-use-styles';
+
+export const useFormNumericInputStyles = createUseStyles(() => ({
+  buttonsAndSubtitleWrapper: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center'
+  },
+  buttonsWrapper: {},
```

Comment:
remove this empty styles.
If you need to hack `conditionalStyle` just wrap it with `[ ]`